CHANGES.rst

```diff
@@ -1,3 +1,9 @@
+0.3.0 (unreleased)
+==================
+
+- Add ``source='github'`` option. Also fix IP lookup error for
+  ``source='astropy'`` option. [#28]
```

Comment:
I didn't mention IPv4/v6 stuff 'cause I don't think users would care. It is transparent to them. FYI.